src/main/java/com/iexec/commons/poco/security/Signature.java

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
Why do we have signature/, tee/ & utils/?

Reply:
`tee` package is required in TaskDescription
Part of `utils` package is required in `chain` package.
`signature` is required by `utils.SignatureUtils` used in `chain` package.

The intention here was to follow and include required packages and classes without heavily modifying things.
Tests and cleanups will come after this first code move (as iso as possible)
Some modifications will require more thorough analyses we don't have the time to perform today.